chromadb/api/models/Collection.py

```diff
@@ -82,14 +83,18 @@ def add(
         )
 
         self._client._add(
-            embedding_set["ids"],
+            cast(IDs, embedding_set["ids"]),
```

Comment:
Why this `cast` ? `ids` on `EmbeddingSet` aren't `Optional`

Reply:
true. updated